scripts/linters/pylint_extensions.py

```diff
@@ -1402,7 +1556,103 @@ def process_module(self, node):
                 elif not any(word in line for word in EXCLUDED_PHRASES):
                     self.add_message(
                         'no-newline-used-at-end', line=line_num + 1)
-                is_docstring = False
+                in_docstring = False
+                current_docstring_section = None
+
+            # All other lines to be checked for returns, raises, yields.
+            elif in_docstring:
+                current_line_indentation = (
+                    len(line_with_spaces) - len(line_with_spaces.lstrip()))
+
+                # Ignore lines that are empty.
+                if len(line) == 0:
+                    continue
+                # If line starts with Returns: , it is the header of a Returns
+                # subsection.
+                elif line.startswith('Returns:'):
+                    current_docstring_section = 'DOCSTRING_SECTION_RETURNS'
```

Comment:
Make these and other similar strings named constants, and refer only to those constants throughout.

Reply:
Done